### ISSUE TITLE
Order Creation: Accessibility → Product announcements can be improved

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#6069]
When using VoiceOver and navigating on a product, I observe these problems:

- VoiceOver doesn't pause. The separate sections (product title, stock) are separated by commas instead of periods. 
- The `•` is read by VoiceOver. This can be removed by adding a custom label. 
- "SKU" is dictated as the word "skew". Not sure if there's anything we can do here. Maybe this isn't worth it to fix right now. 

<img src="https://user-images.githubusercontent.com/198826/152431057-ab2bd6a0-e0f0-4d0d-92b1-c6386b2dcedf.png" width="300">

**Comment by rachelmcr** [comment]
> "SKU" is dictated as the word "skew". Not sure if there's anything we can do here. Maybe this isn't worth it to fix right now.

As mentioned in Slack, this is one way to pronounce "SKU" at least in US English so it should be ok to leave as-is.
